lib/Cake/Model/Datasource/CakeSession.php

```diff
@@ -218,6 +217,9 @@ public static function start() {
  * @return bool True if session has been started.
  */
 	public static function started() {
+		if (PHP_VERSION >= 5.4) {
+			return isset($_SESSION) && (session_status() === PHP_SESSION_ACTIVE);
```

Comment:
Instead of sniffing the version, could we check for the existence of `session_status()`?